src/Evolution/Executables/Cce/CMakeLists.txt

```diff
@@ -16,11 +16,11 @@ set(LIBS_TO_LINK
   Utilities
   )
 
-function(add_cce_executable EXECUTABLE BOUNDARY_COMPONENT)
+function(add_cce_executable EXECUTABLE BOUNDARY_COMPONENT SYSTEM)
   add_spectre_executable(
     ${EXECUTABLE}
     EXCLUDE_FROM_ALL
-    CharacteristicExtract.cpp
+    ${SYSTEM}
```

Comment:
Maybe do
```cpp
${SYSTEM}.cpp
```

so you don't always have to add the `.cpp` in the function calls below.